api/utils/ira_calculator.py

```diff
@@ -0,0 +1,37 @@
+class IraCalculator:
+    def __init__(self) -> None:
+        self.mencaoMap = {
+            'SS': 5,
+            'MS': 4,
+            'MM': 3,
+            'MI': 2,
+            'II': 1,
+            'SR': 0,
+        }
```

Comment:
Será que não seria interessante a utilização do [enum](https://www.geeksforgeeks.org/enum-in-python/) nesse caso? 
O que acham? @lucasqueiroz23 @GabrielCastelo-31

Reply:
acredito que a verificação da entrada é mais fácil se esse objeto se mantiver como um dict.